services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -4394,7 +4410,7 @@ private void internalUpdateStore(String clusterName, String storeName, UpdateSto
 
       if (currentVersion.isPresent()) {
         if (childRegionOnlyConfigUpdateAllowed) {
```

Comment:
can we reuse the variable `onlyParentRegionFilter` ? Also simply the logic something  like

`if (currentVersion.isPresent()) {
        setStoreCurrentVersion(clusterName, storeName, currentVersion.get(), onlyParentRegionFilter);
 }`

Reply:
Makes sense. Updated